client/js/widgets/scoreboard.js

```diff
@@ -48,6 +48,54 @@ class Scoreboard extends Widget {
     return p;
   }
 
+  async click(mode='respect') {
+    if(!await super.click(mode)) {
+      const scoreProperty = this.get('scoreProperty');
+      const seats = this.getIncludedSeats();
+      let players = Object.keys(seats).map(function(s) { return { text: s, value: s }; });
+      if(Array.isArray(seats))
+        players = seats.map(function(s) { return { value: s.get('id'), text: s.get('player') }; });
+
+      const rounds = this.getRounds(seats, scoreProperty, 1).map(function(r, i) { return { text: r, value: i+1 }; });
+
+      if(!players.length || !rounds.length)
```

Comment:
`rounds.length` will always be at least 1 because of the way `getRounds` now works. This is why editing totals-only scoreboards doesn't work properly.